vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -43,7 +43,7 @@ packages:
         - libssl-dev
     pip3:
      - python-apt
-     - openshift==0.11.2
+     - kubernetes==17.17.0
```

Comment:
is this an ansible k8s module version?

Reply:
Ansible `k8s 2.1.0 `requires pip package `kubernetes>=12.0.0 `

Reply:
Can you please update the PR description with that note and if possible reference to source info, otherwise it in unclear why do we need that package. Thanks.